data/risingwave/models/10_bronze/energy/energy.yml

```diff
@@ -0,0 +1,63 @@
+version: 2
+models:
+  - name: battery_dc_converter_power_updates
+    description: Voltage and current value updates of the battery DC converters. 
+    columns:
+      - name: topic
+        description: '{{ doc("topic") }}'
+        data_tests:
+            - not_null
+      - name: time
+        description: '{{ doc("zero_timestamp") }}'
+        data_tests:
+            - not_null
+      - name: voltage_a
+        description: '{{ doc("voltage_at_side") }} A.'
+      - name: voltage_a_timestamp
+        description: '{{ doc("timestamp") }}'
```

Comment:
For these values I'd either just leave out the description or just say directly "Timestamp at which the voltage A value was reported". DRYing this doesn't add much